resources/views/training/create.blade.php

```diff
@@ -54,9 +54,35 @@ class="form-control @error('student') is-invalid @enderror"
                             <span class="text-danger">{{ $errors->first('training_area') }}</span>
                         @enderror
                     </div>
+
+                    <div class="form-group">
+                        <label class="my-1 mr-2" for="typeSelect">Training type</label>
+                        <select id="typeSelect" name="type" class="custom-select my-1 mr-sm-2 @error('type') is-invalid @enderror" @change="onChange($event)">
+                            <option selected disabled>Choose training type</option>
+                            @foreach($types as $id => $data)
+                                @if($id == 1)
```

Comment:
Could you explain what this check is for, ain't all first occurrences selected implicitly by browser?

Reply:
It's there to have 'Standard' preselected. At least, Chromium or Firefox don't select the first option automatically and instead will have nothing selected without this.